src/runtime_src/core/common/shim/hwctx_handle.h

```diff
@@ -9,8 +9,10 @@
 #include "core/common/shim/shared_handle.h"
 
 #include "xrt/xrt_hw_context.h"
+#include "xcl_graph.h"
```

Comment:
why?
you may need to just include xrt_graph.h not xcl_graph.h

Reply:
I think we should create shim/graph_handle.h in anticipation that a graph_handle could be more than a void*.   For the time being create shim/graph_handle.h and simply do
```
namespace xrt_core {
using graph_handle = void*;
}
```
Then #include graph_handle.h from hwctx_handle.h and return graph_handle in open_graph.

Reply:
That is good suggestion @stsoe . 
@SravanKumarAllu-xilinx , Lets include xrt_graph.h in this PR. Create a separate PR for converting current graph handles with newly created shim handles.